data/templates/news-article.mst

```diff
@@ -1,37 +1,55 @@
-{{#extra-header-information}}
 <div class="extra-header">
-    <div class="context"><p>{{{context}}}</p></div>
+    <div class="context">
+        <a class="eos-show-link" id="context-link"></a>
+    </div>
     <div class="extra-header-right">
-        {{#date-published}}
-        <p class="date-published">{{.}}</p>
-        {{/date-published}}
-        {{#source-link}}
+        <p class="date-published" id='date-published'></p>
         <p class="source-link">
-            {{{.}}}
-            {{#author}}
-            &mdash; {{.}}
-            {{/author}}
+            <a class="eos-show-link" id="source-link">Prensalibre.com</a>
+            &mdash; <span id="authors"></span>
         </p>
-        {{/source-link}}
     </div>
 </div>
-{{/extra-header-information}}
+
 <section id="inside-content" class="section">
     <div>{{{body-html}}}</div>
     <hr>
-    {{#disclaimer}}
     <div class="disclaimer">
-        <p>{{{.}}}</p>
+        <p>
+        Leer más en <a class="eos-show-link" id="disclaimer-link">Prensalibre.com</a><br>
+        <a class="eos-modal-link" href="#modal"> Aviso Legal y Política de Propiedad Intelectual </a>
+        </p>
     </div>
-    {{/disclaimer}}
     <div class="eos-modal" id="modal">
         <a class="eos-modal-close-background" href="#close"></a>
         <div>
             <a class="eos-modal-close-button" href="#close"> &times </a>
```

Comment:
today I learned that the semicolon is optional on character entities for historical reasons :stuck_out_tongue_closed_eyes: